### PR TITLE
build: upgrade compile docker & support `CONFIG CLAUSE`

### LINKED ISSUE #627: ci: openmldb-batch deploy failed on macos version
https://github.com/4paradigm/OpenMLDB/runs/4075637707?check_suite_focus=true
image

It seems like, set openmldb-batch version to xxx-macos-xx failed.

### LINKED ISSUE #676: ci: refactor Github jobs, merge HybridSE and OpenMLDB workflow
1. Merge the `hybridse-ci` workflow into `cicd` workflow. 
- `hybridse-ci`'s `test report` and `release` action can be removed.
- merge serval `prepare_release.sh` script
   hint: tweak cpp and python version should removed, instead, create a bump version commit directly before release

2. Better workflow style
- #495 

openmldb-jobs drawio

### PR DESCRIPTION
**Pull request by aceforeverd**
- upgrade zetasql to v0.2.6
- upgrade compile image to `hybridsql:0.4.1`
- emit 'make thirdparty' from workflow
- fix the java sdk deploy #897
- suuport `CONFIG CLAUSE`: #907

### CODE REVIEW

**Review by zhanghaohit — CHANGES_REQUESTED** [review]
You may want to replace all the occurrences of `hybridsql:0.4.0` to `hybridsql:0.4.1`, e.g., docs/en/compile.md, hybridse/README.md, and all the others.

File: steps/init_env.sh [hunk 1]
```diff
@@ -17,6 +17,8 @@
 # init_env.sh
 set -eE -x
 
+echo -e "${RED}this script is deprecated, please run 'make thirdparty' in top directory instead${NC}"
```
Comment (dl239) [inline steps/init_env.sh:20]: delete this unused file directly?

Comment (aceforeverd) [reply steps/init_env.sh:20]: https://github.com/4paradigm/OpenMLDB/blob/main/.github/workflows/integration-test-src.yml not upgraded @wuyou10206

File: hybridse/src/vm/transform.cc [hunk 2]
```diff
@@ -795,6 +795,9 @@ Status BatchModeTransformer::TransformUnionOp(const node::UnionPlanNode* node,
     PhysicalUnionNode* union_op = nullptr;
     CHECK_STATUS(
         CreateOp<PhysicalUnionNode>(&union_op, left, right, node->is_all));
+    if (node->config_options_ != nullptr) {
+        union_op->config_options_ = node->config_options_;
```
Comment (aceforeverd) [inline hybridse/src/vm/transform.cc]: union query not support

**Comment by aceforeverd** [comment]
[github workflow reorganization](https://trello.com/c/upawl5mF/8-github-workflow-reorganization)

**Comment by zhanghaohit** [comment]
> You may want to replace all the occurrences of `hybridsql:0.4.0` to `hybridsql:0.4.1`, e.g., docs/en/compile.md, hybridse/README.md, and all the others.

Pls see whether it is necessary to update these four files.
```
integration-test-src.yml
integration-test-pkg.yml
hybridse/README.md
hybridse/README-zh.md
```

**Comment by aceforeverd** [comment]
> 

nice point.
hybridse's README is out-of-date, it's better to just delete it in case confusing @lumianph @jingchen2222

**Comment by aceforeverd** [comment]
> > You may want to replace all the occurrences of `hybridsql:0.4.0` to `hybridsql:0.4.1`, e.g., docs/en/compile.md, hybridse/README.md, and all the others.
> 
> Pls see whether it is necessary to update these four files.
> 
> ```
> integration-test-src.yml
> integration-test-pkg.yml
> hybridse/README.md
> hybridse/README-zh.md
> ```

nice point.
hybridse's README is out-of-date, it's better to just delete it in case confusing @lumianph @jingchen2222

**Comment by aceforeverd** [comment]
update java_sdk md

**Review by zhanghaohit — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: hybridse/src/vm/transform.cc (hunk 2 above)
Comment (aceforeverd) [reply hybridse/src/vm/transform.cc]: conflicts with #796, will rm it

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**